kolibri/core/sqlite/management/commands/vacuumsqlite.py

```diff
@@ -0,0 +1,49 @@
+import logging as logger
```

Comment:
Ahaha, first line and I have an objection - sorry :) Why alias this and then assign `logging` further down? I would prefer if we could have a strict, consistent logging pattern. So just:

```
import logging
logger = logging.getLogger(__name__)
```

Reply:
Initially I coded it as you say, but later I changed it to this way for consistency reasons.
Kolibri uses always `logging.info...` not `logger.info`

Reply:
Huh? Oh, do we have a rogue pattern somewhere? Could you ref that? Sounds like a cleanup issue then...

Reply:
@benjaoming  everywhere, just do a grep for it in the code. All kolibri code is that way.